internal/policies/client.go

```diff
@@ -204,17 +211,20 @@ func (f *Client) groupPoliciesByGVR(ctx context.Context, policies []policiesv1.P
 							continue
 						}
 
-						if !isAuditable(policy) {
+						if !policy.GetBackgroundAudit() {
 							skippedPolicies[policy.GetUniqueName()] = struct{}{}
+							log.Debug().Str("policy", policy.GetUniqueName()).Msg("the policy has backgroundAudit set to false, skipping!")
+
+							continue
+						}
 
-							log.Debug().Str("policy", policy.GetUniqueName()).
-								Bool("backgroundAudit", policy.GetBackgroundAudit()).
-								Bool("active", policy.GetStatus().PolicyStatus == policiesv1.PolicyStatusActive).
-								Bool("create", isCreateActionPresentWithoutAllResources(policy)).
-								Msg("not auditable policy, skipping!")
+						if policy.GetStatus().PolicyStatus != policiesv1.PolicyStatusActive {
```

Comment:
this could also be moved outside of the `rules` for , akin to `!isCreateActionPresentWithoutAllResources(policy)`.

Reply:
the reason why it is here is because we need the check the above condition if the policy targets resource with the right scope, otherwise policies that are not active but targeting for instance namespaced resource when scanning a namespace will be considered as skipped as well

Reply:
then, maybe we need a `break`? If not, it seems to be that we will append the policy several times per resource and version.